### ISSUE TITLE
SSH Keys

### ISSUE THREAD

**Issue by viglesiasce** [danisla/terraform-google-k8s-gce#2]
Add SSH keys so at least the NAT box can access the master. In the future we should setup a separate jumpbox alongside the NAT for this purpose.

**Comment by danisla** [comment]
I agree, the module should accept some list of public keys that are allowed.

Today, per the docs in the [README for the example](https://github.com/danisla/terraform-google-examples/blob/master/example-k8s-gce/README.md#run-terraform), you can use the NAT gateway as a jumpbox to get to the master like below.

```
gcloud compute ssh --ssh-flag="-A" \
  $(gcloud compute instances list --filter=nat-gateway-us-central --uri) \
  -- ssh $(gcloud compute instances list --filter='name~k8s-.*master.*' --format='get(name)')
```

**Comment by viglesiasce** [comment]
I'd like to have a separate jumpbox from the NAT since all traffic will be heading through the NAT box.

What I did was setup a jumpbox in the same network then used sshuttle to connect to it in order to hop to the master. As for SSH keys, I just added mine to the project metadata.
